backend/app/static/js/form.js

```diff
@@ -25,6 +25,9 @@ $(document).ready(
             $("#background-input").css("display", "block");
             $("input[name='img-default']").val("user_defined.png");
         });
+        $("#text-color").click(function () {
+            $("#text-fill-input").css("display", "block");
```

Comment:
This is not required and should be removed.

Reply:
why, how will I get a color picker then ?

Reply:
Ok, i think, you meant in backend folder, I don't need to change javascript code, only in front - end folder its required. Isn't it?

Reply:
Yes. We don't require a Change in the backend folder @rupav